### PR TITLE
Feature/dx 2038 create section for old implementations

### PR DESCRIPTION
**Pull request by arebra**
Renamed sections and renamed in text

### CODE REVIEW

File: _includes/payer-aware-payment-menu.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ all payment instruments provided by Swedbank Pay support Payer Awareness today."
 
 ## BYO Payment Menu
 
-Payment Menu is versatile and can be configured in such a way that it functions
-like a single payment instrument. In such configuration, it is easy to Bring
-Your Own Payment Menu, i.e. building a customized payment menu in our own user
-interface.
+Payment Menu v2 is versatile and can be configured in such a way that it
+functions like a single payment instrument. In such configuration, it is easy to
+Bring Your Own Payment Menu, i.e. building a customized payment menu in our own
```
Comment (m-lilja) [inline _includes/payer-aware-payment-menu.md:29]: Is "our" correct? Or should it be "your"?

`In such configuration, it is easy to Bring Your Own Payment Menu, i.e. building a customized payment menu in **our** own user interface.`

File: checkout-v3/migration-guide.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ Checkout v3 provides a more stream-lined, refined checkout experience for both
 the payer and the merchant. The response models have been reduced in size and
 complexity to simplify integration.
 
-Where Checkout v2 and Payment Instruments were offered more as a mix and match
-package, Checkout v3 is more conveniently divided into four distinct packages:
-_Starter_, _Business_, _Enterprise_, and _Payments Only_.
+Where Checkout v2 and Payment Instruments v1 were offered more as a mix and
+match package, Checkout v3 is more conveniently divided into four distinct
+packages: _Starter_, _Business_, _Enterprise_, and _Payments Only_.
```
Comment (m-lilja) [inline checkout-v3/migration-guide.md:17]: Should we still talk about Starter & Business?

File: _includes/alert-callback-url.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {% capture documentation_section %}{% include utils/documentation-section.md %}{% endcapture %}
-{% assign callback_href = "/payment-instruments/" | append: documentation_section | append: "/features/core/callback" %}
+{% assign callback_href = "/old-implementations/payment-instruments-v1/" | append: documentation_section | append: "/features/core/callback" %}
```
Comment (m-lilja) [inline _includes/alert-callback-url.md:2]: Should we call it "old-implementations"?  Or should we use the term "deprecated-implementations"?

File: checkout-v3/migration-guide.md (hunk 2 above)
Comment (arebra) [reply checkout-v3/migration-guide.md:17]: This is a in the migration guide, which is hidden from the documentation and has to be rewritten completely, so I have let it be for now. Will be removed :)

File: _includes/payer-aware-payment-menu.md (hunk 1 above)
Comment (arebra) [reply _includes/payer-aware-payment-menu.md:29]: Seems to be a typo. Your makes more sense! Good catch 👍

**Review by m-lilja — APPROVED** [review]
LGTM

**Pull request merged**